BAZProjectC1/Components/BasicInfoMovie/BasicInfoMovie.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BasicInfoMovie.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import UIKit
+
+class BasicInfoMovie: UIView {
+    var viewBackgroundColor: UIColor? = .clear
+    private lazy var backView: UIView = {
+        let backView = UIView(frame: self.bounds)
+        backView.layer.cornerRadius = RounderBorderStyleForView.rounded.rawValue
+        return backView
+    }()
+    
+    public convenience init(frame:CGRect,viewBackgroundColor: UIColor){
```

Comment:
Es necesario que este inicializador sea publico? Podría ser internal?

Ten cuidado con los espacios:
```suggestion
    public convenience init(frame:CGRect,viewBackgroundColor: UIColor) {
```